angular-chart.js

```diff
@@ -2,7 +2,7 @@
   'use strict';
   if (typeof exports === 'object') {
     // Node/CommonJS
-    module.exports = factory(require('angular'), require('Chart.js'));
+    module.exports = factory(angular ? angular : require('angular'), require('Chart.js'));
```

Comment:
That would create an error if angular is not defined. Use `window.angular` or `typeof angular`.